osquery/main/windows/main.cpp

```diff
@@ -393,14 +316,26 @@ int main(int argc, char* argv[]) {
        static_cast<LPSERVICE_MAIN_FUNCTION>(osquery::ServiceMain)},
       {nullptr, nullptr}};
 
+  auto shutdown = ([]() {
+    // The event only gets initialized in the entry point of the service. Child
+    // processes and those run from the commandline will have kStopEvent as a
+    // nullptr
+    if (osquery::kStopEvent != nullptr) {
```

Comment:
Wait, I just missed a call-site for `startOsquery`: https://github.com/theopolis/osquery/blob/01c95b6c41cf857f4770540ecf9e4a60b707ae3d/osquery/main/windows/main.cpp#L297 that needs the `shutdown` lambda.